app/addons/custom_carriers/controllers/backend/addons.post.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/***************************************************************************
+ *                                                                          *
+ *   (c) 2004 Vladimir V. Kalynyak, Alexey V. Vinokurov, Ilya M. Shalnev    *
+ *                                                                          *
+ * This  is  commercial  software,  only  users  who have purchased a valid *
+ * license  and  accept  to the terms of the  License Agreement can install *
+ * and use this program.                                                    *
+ *                                                                          *
+ ****************************************************************************
+ * PLEASE READ THE FULL TEXT  OF THE SOFTWARE  LICENSE   AGREEMENT  IN  THE *
+ * "copyright.txt" FILE PROVIDED WITH THIS DISTRIBUTION PACKAGE.            *
+ ****************************************************************************/
+
+use Tygh\Enum\NotificationSeverity;
+use Tygh\Enum\ObjectStatuses;
+use Tygh\Enum\YesNo;
+use Tygh\Settings;
+
+defined('BOOTSTRAP') or die('Access denied');
+
+/**
+ * @var string $mode
+ */
+
+if (
+    $mode === 'update'
+    && $_SERVER['REQUEST_METHOD'] === 'POST'
+    && $_REQUEST['addon'] === 'custom_carriers'
+) {
+    $new_carriers = [];
+    for ($i = 1; $i <= 3; $i++) {
+        $str_i = (string) $i;
+        $carrier_name = Settings::instance()->getValue('carrier_name_' . $str_i, '');
+        if (!empty($carrier_name)) {
+            $carrier_name = mb_strtolower($carrier_name);
```

Comment:
```suggestion
            $carrier_name = fn_strtolower($carrier_name);
```

Но в БД останется все то же значение в верхнем регистре.